examples/plot_mne_example.py

```diff
@@ -48,12 +48,12 @@
 # First, we will load the example dataset from MNE, and have a quick look at the data.
 #
 # The MNE sample dataset is a combined MEG/EEG recording with an audiovisual task, as
-# described `here <https://martinos.org/mne/stable/manual/sample_dataset.html>`_.
+# described `here <https://mne.tools/0.11/manual/datasets_index.html#sample>`_.
```

Comment:
This link is to a specific, and old, version of MNE. We should link to the stable release. 
I think that would be this link:
https://mne.tools/stable/overview/datasets_index.html?#sample

Reply:
Updated link.